### PR TITLE
Improves Nova 5 Support

### PR DESCRIPTION
**Pull request by crynobone**
* use `laravel/nova-devtool` to improve local development
* use `npm` instead of `yarn` (default with Laravel Nova 4 and 5)
* format js syntax with `npm run format`
* use Nova Workbench to handle tests and `composer run serve` to preview the package.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -10,11 +10,6 @@
...
   </testsuites>
-    <php>
-        <env name="DB_CONNECTION" value="mysql"/>
-        <env name="DB_USERNAME" value="root"/>
-        <env name="DB_DATABASE" value="nova_tags_field"/>
-        <env name="DB_HOST" value="127.0.0.1"/>
-        <env name="DB_PORT" value="3306"/>
-    </php>
+  <php>
+  </php>
```
Comment (crynobone) [inline phpunit.xml.dist:14]: Use `sqlite` by default. Run locally using `composer test`

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -9,14 +9,14 @@ jobs:
...
-                include:
-                    -   laravel: 9.*
-                        testbench: 7.*
-                    -   laravel: 8.*
-                        testbench: 6.*
+                exclude:
+                    -   php: 8.4
+                        laravel: 10.*
+                    -   php: 8.0
+                        laravel: 11.*
```
Comment (crynobone) [inline .github/workflows/run-tests.yml:19]: Fix to run test on correct versions.

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -49,12 +49,15 @@ jobs:
             -   name: Install dependencies
                 run: |
                     composer config "http-basic.nova.laravel.com" "${{ secrets.NOVA_USERNAME }}" "${{ secrets.NOVA_PASSWORD }}"
-                    composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-                    composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction
+                    composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --with="laravel/framework:${{ matrix.laravel }}"
```
Comment (crynobone) [inline .github/workflows/run-tests.yml:52]: Testbench now add conflicts and we no longer need to specify both laravel and testbench version here.

File: .github/workflows/run-tests.yml [hunk 4]
```diff
@@ -49,12 +49,15 @@ jobs:
...
                   COMPOSER_AUTH: ${{ secrets.COMPOSER_AUTH }}
 
             -   name: Execute tests
                 run: vendor/bin/phpunit
                 env:
+                    DB_CONNECTION: 'mysql'
+                    DB_USERNAME: 'root'
+                    DB_DATABASE: 'nova_tags_field'
+                    DB_HOST: '127.0.0.1'
                     DB_PORT: ${{ job.services.mysql.ports[3306] }}
```
Comment (crynobone) [inline .github/workflows/run-tests.yml:63]: explicitly use mysql on GA.

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**